PodcastGenerator/core/feed_generator.php

```diff
@@ -51,6 +51,9 @@ function generateRSS()
     if ($config['itunes_category[2]'] != '' || $config['itunes_category[1]'] == 'null') {
         $feedhead .= '		<itunes:category text="' . htmlspecialchars($config['itunes_category[2]']) . '"></itunes:category>' . "\n";
     }
+    if ($config['websub_server'] && $config['websub_server'] != 'null') {
```

Comment:
`if ($config['websub_server'] != "")` is probably the most elegant way no need for `null` checking please as other parts of the code don't do that either.

Reply:
So `$config['websub_server'] != '' && $config['websub_server'] != 'null'` then?

Reply:
No just `$config['websub_server'] != ''`

Reply:
Changed in be0e796